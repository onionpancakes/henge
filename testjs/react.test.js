import React from 'react';
import renderer from 'react-test-renderer';

it("lowercase dot access are user components", () => {
    let foo = {
        bar() {
            return <div>"bar"</div>;
        }
    };
    let component = renderer.create(
        <foo.bar></foo.bar>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
