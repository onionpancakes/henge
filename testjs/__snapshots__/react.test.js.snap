// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lowercase dot access are user components 1`] = `
<div>
  "bar"
</div>
`;
